torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10425,6 +10425,14 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            supports_out=False),
+    OpInfo('unsafe_chunk',
+           dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16, torch.chalf),
+           sample_inputs_func=sample_inputs_chunk,
+           check_batched_forward_grad=False,
```

Comment:
gradcheck uses legacy vmap implementation for batched forward and `unsafe_chunk` uses `unsafe_split` which doesn't have a batching rule for legacy vmap. (`unsafe_split` OpInfo also sets this to False)